packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -102,10 +102,34 @@ describe('packages/date-picker', () => {
         expect(menuContainerEl).not.toBeInTheDocument();
       });
 
+      test('menu is initially closed when rendered with `initialOpen` and `disabled`', async () => {
```

Comment:
Super nitty, but more specific tests should go after more general ones (i.e. this one should go after the basic `initialOpen` test)